cpp/apidoc/Windows.md

```diff
@@ -41,7 +41,7 @@ conda config --add channels conda-forge
 Now, you can bootstrap a build environment
 
 ```shell
-conda create -n arrow-dev cmake git boost-cpp flatbuffers rapidjson cmake thrift-cpp snappy zlib brotli gflags lz4-c zstd -c conda-forge
+conda create -n arrow-dev python=3 cmake git boost-cpp flatbuffers rapidjson cmake thrift-cpp snappy zlib brotli gflags lz4-c zstd -c conda-forge
```

Comment:
Which dependency was causing Python to get installed? At least when these docs were written, none of these libraries had a Python dependency

Reply:
Hmm, I can't actually reproduce this - I might have had an old version of conda installed or something - it definitely was picking up python 2.7, but with a clean Miniconda install isn't.  I'll revert.
```
"C:\Users\chris.bartak\AppData\Local\Continuum\miniconda3\Scripts\conda.exe" create -n arrow-dev-test cmake git boost-cpp flatbuffers rapidjson cmake thrift-cpp snappy zlib brotli gflags lz4-c zstd -c conda-forge
Fetching package metadata ...............
Solving package specifications: .

Package plan for installation in environment C:\Users\chris.bartak\AppData\Local\Continuum\miniconda3\envs\arrow-dev-test:

The following NEW packages will be INSTALLED:

    boost-cpp:       1.66.0-vc14_1 conda-forge [vc14]
    brotli:          0.6.0-vc14_0  conda-forge [vc14]
    ca-certificates: 2017.11.5-0   conda-forge
    cmake:           3.10.0-4      conda-forge
    flatbuffers:     1.7.1-vc14_0  conda-forge [vc14]
    gflags:          2.2.0-vc14_1  conda-forge [vc14]
    git:             2.14.2-3      conda-forge
    lz4-c:           1.8.0-vc14_0  conda-forge [vc14]
    openssl:         1.0.2n-vc14_0 conda-forge [vc14]
    rapidjson:       1.1.0-0       conda-forge
    snappy:          1.1.7-vc14_1  conda-forge [vc14]
    thrift-cpp:      0.10.0-vc14_1 conda-forge [vc14]
    vc:              14-0          conda-forge
    vs2015_runtime:  14.0.25420-0  conda-forge
    zlib:            1.2.11-vc14_0 conda-forge [vc14]
    zstd:            1.3.2-vc14_1  conda-forge [vc14]
```